Library/Formula/av98.rb

```diff
@@ -0,0 +1,23 @@
+class Av98 < Formula
+  desc "AV-98 is an experimental client for the Gemini protocol"
+  homepage "https://tildegit.org/solderpunk/AV-98"
+  url "https://tildegit.org/solderpunk/AV-98/archive/v1.0.1.tar.gz"
```

Comment:
Hmm, this is serving a 500. Is there a good mirror somewhere?

Reply:
Thanks for the heads up, it seems like the Tilde services have been experiencing issues in the last week, similar problem with another of their services. I'll see if I can find a mirror and get someone to poke at it.

Reply:
Looks like it's still down, unfortunately. :/